File: src/main/webapp/osca/common/domain/Page/Rubriques/SousRubriquesPage/SousRubriquesPage.ts
import { FeuilleDeRouteDevPage } from './Dev/FeuilleDeRouteDev';
import { FeuilleDeRouteFoncPage } from './Fonc/FeuilleDeRouteFoncPage';
import { FeuilleDeRouteUXPage } from './UX/FeuilleDeRouteUXPage';
import { Maquette } from './Maquette';

export interface SousRubriquesPage {
    keySousRubriquesPage: string;
    feuilleDeRouteDevPage: FeuilleDeRouteDevPage[];
    feuilleDeRouteUXPage: FeuilleDeRouteUXPage[];
    feuilleDeRouteFoncPage: FeuilleDeRouteFoncPage[];
    maquette: Maquette[];
}

File: src/main/java/com/soprasteria/osca/domain/page/rubriques/sousrubriques/SousRubriquesPage.java
package com.soprasteria.osca.domain.page.rubriques.sousrubriques;

import com.soprasteria.osca.domain.maquette.Maquette;
import com.soprasteria.osca.domain.page.rubriques.sousrubriques.dev.FeuilleDeRouteDevPage;
import com.soprasteria.osca.domain.page.rubriques.sousrubriques.fonc.FeuilleDeRouteFoncPage;
import com.soprasteria.osca.domain.page.rubriques.sousrubriques.ux.FeuilleDeRouteUXPage;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class SousRubriquesPage {

    private String keySousRubriquesPage;
    private List<FeuilleDeRouteDevPage> feuilleDeRouteDevPage;
    private List<FeuilleDeRouteUXPage> feuilleDeRouteUXPage;
    private List<FeuilleDeRouteFoncPage> feuilleDeRouteFoncPage;
    private List<Maquette> maquette;

}
